test-worker/Context/Do/user/addGroup.ts

```diff
@@ -0,0 +1,38 @@
+import * as gracely from "gracely"
+import * as http from "cloudly-http"
+import * as model from "../../../model"
+import { Context } from "../Context"
+import { router } from "../router"
+
+export async function addGroup(request: http.Request, context: Context): Promise<http.Response.Like | any> {
+	let result: gracely.Result | gracely.Error
+	const state = context.state
+	const id = request.parameter.id
+	const newGroups: string[] | undefined = await request.body
+
+	if (gracely.Error.is(state))
+		result = state
+	else if (!id)
+		result = gracely.server.backendFailure("id missing in database query.")
+	else if (newGroups == undefined || !Array.isArray(newGroups))
+		result = gracely.client.invalidContent("string", "Body does not contain a valid group")
+	else if (request.body && request.parameter.group)
```

Comment:
Some of this logic is redundant due to the fact that we removed `request.parameter.group`.